<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <group ns="tj2">
        <node pkg="tj2_turret" type="tj2_turret_node.py" name="tj2_turret" output="screen" required="false">
            <remap from="amcl_pose" to="/amcl_pose"/>
            <param name="time_of_flight_file_path" value="$(find tj2_turret)/config/time_of_flight.csv"/>
            <param name="recorded_data_file_path" value="$(find tj2_turret)/config/recorded_data.csv"/>
            <param name="probability_hood_up_map_path" value="$(find tj2_turret)/maps/$(env ROS_MAP_NAME)-up.yaml"/>
            <param name="probability_hood_down_map_path" value="$(find tj2_turret)/maps/$(env ROS_MAP_NAME)-down.yaml"/>
            <param name="enable_shot_correction" value="true"/>
            <param name="enable_shot_probability" value="false"/>
            <param name="enable_limelight_fine_tuning" value="false"/>
            <param name="turret_cosmic_ray_compensation_degrees" value="0.0"/>
        </node>
    </group>
</launch>
